Library/Formula/git-multipush.rb

```diff
@@ -1,17 +1,17 @@
 class GitMultipush < Formula
-  homepage 'http://code.google.com/p/git-multipush/'
-  url 'https://git-multipush.googlecode.com/files/git-multipush-2.3.tar.bz2'
-  sha1 'a53f171af5e794afe9b1de6ccd9bd0661db6fd91'
+  homepage "https://github.com/gavinbeatty/git-multipush"
+  url "https://git-multipush.googlecode.com/files/git-multipush-2.3.tar.bz2"
+  sha256 "1f3b51e84310673045c3240048b44dd415a8a70568f365b6b48e7970afdafb67"
 
   devel do
-    url 'https://github.com/gavinbeatty/git-multipush/archive/git-multipush-v2.4.rc2.tar.gz'
-    sha1 '7179bc729a1dee76a76e3c91935f524911037313'
-    version '2.4-rc2'
+    url "https://github.com/gavinbeatty/git-multipush/archive/git-multipush-v2.4.rc2.tar.gz"
+    sha256 "999d9304f322c1b97d150c96be64ecde30980f97eaaa9d66f365b8b11894c46d"
+    version "2.4-rc2"
   end
 
-  head 'https://github.com/gavinbeatty/git-multipush.git'
+  head "https://github.com/gavinbeatty/git-multipush.git"
 
-  depends_on 'asciidoc' => :build
+  depends_on "asciidoc" => :build
 
   def install
     system "make" if build.head?
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!